### ISSUE TITLE
merging of Pullrequest

### ISSUE THREAD

**Issue by NPadrutt** [michelsalib/Gi7#76]
How do you guys plan to handle the stuff with the pull request? Everyone merges his own pull requests as soon as he think it's done, a single person who checkes all pull request or something like each pull request is be merged by someone other than the creator?

**Comment by RandomlyKnighted** [comment]
That's a good question, we should definitely be consistent. I think each pull request should be merged by someone other than the creator that way everything can be reviewed before being merged (in order to help catch potential problems).

**Comment by NPadrutt** [comment]
This is my favorite option as well.

@AlbertoMonteiro @michelsalib what do you mean?

**Comment by AlbertoMonteiro** [comment]
I agree, and we should write tests to have more guarantee when writing new code and doesn't break existent one.

**Comment by RandomlyKnighted** [comment]
Is anyone good at writing tests? I know I could still use some practice at it.

**Comment by NPadrutt** [comment]
Well, I can write tests. But I wouldn't call me an expert. 
But let's make an own issue for the unit tests discussion.

To come back to the topic here: pull request have to be merged by someone other than the creator -> all fine with that?

**Comment by RandomlyKnighted** [comment]
Sounds good!

**Comment by AlbertoMonteiro** [comment]
Ok for me
